README.md

```diff
@@ -400,6 +402,38 @@ Here are some examples of conditions you can use:
 > 💡 Use `pat` only when you need to. `[STATUS] == pat(2*)` is a lot more expensive than `[STATUS] < 300`.
 
 
+### Announcements
+System-wide announcements allow you to display important messages at the top of the status page. These can be used to inform users about planned maintenance, ongoing issues, or general information.
```

Comment:
Do you think it's worth mentioning that one can edit the config and it'll gracefully propagate after 30 seconds thanks to listenToConfigurationFileChanges? Because at first glance, it seems like one might need to restart the server(?)

I might be overthinking it though.

Reply:
Nah, because either way, most people won't directly modify the config file that's deployed in production, they'll deploy the changes using something like terraform, which makes the automatic propagation point kinda moot